projects/api-client/src/api/artifact.service.ts

```diff
@@ -137,30 +157,96 @@ export class ArtifactService {
+     * @param buildId Build ID
+     * @param limit Number of results to return. No limiting is applied if empty (&#x60;?limit&#x3D;&#x60;) or non-positive (&#x60;?limit&#x3D;0&#x60;). Required if &#x60;offset&#x60; is used.
+     * @param offset Skipped results, where 0 means from the start.
+     * @param orderby Sorting orders. Takes the property name followed by either &#39;asc&#39; or &#39;desc&#39;. Can be specified multiple times for more granular sorting. Defaults to &#x60;?orderby&#x3D;artifactId desc&#x60;
+     * @param name Filter by verbatim artifact name.
+     * @param fileName Filter by verbatim artifact file name.
+     * @param nameMatch Filter by matching artifact name. Cannot be used with &#x60;name&#x60;.
+     * @param fileNameMatch Filter by matching artifact file name. Cannot be used with &#x60;fileName&#x60;.
+     * @param match Filter by matching on any supported fields.
+     * @param observe set whether or not to return the data Observable as the body, response or events. defaults to returning the body.
+     * @param reportProgress flag to report request and response progress.
+     */
+    public getBuildArtifactList(buildId: number, limit?: number, offset?: number, orderby?: Array<string>, name?: string, fileName?: string, nameMatch?: string, fileNameMatch?: string, match?: string, observe?: 'body', reportProgress?: boolean): Observable<ResponsePaginatedArtifacts>;
+    public getBuildArtifactList(buildId: number, limit?: number, offset?: number, orderby?: Array<string>, name?: string, fileName?: string, nameMatch?: string, fileNameMatch?: string, match?: string, observe?: 'response', reportProgress?: boolean): Observable<HttpResponse<ResponsePaginatedArtifacts>>;
+    public getBuildArtifactList(buildId: number, limit?: number, offset?: number, orderby?: Array<string>, name?: string, fileName?: string, nameMatch?: string, fileNameMatch?: string, match?: string, observe?: 'events', reportProgress?: boolean): Observable<HttpEvent<ResponsePaginatedArtifacts>>;
+    public getBuildArtifactList(buildId: number, limit?: number, offset?: number, orderby?: Array<string>, name?: string, fileName?: string, nameMatch?: string, fileNameMatch?: string, match?: string, observe: any = 'body', reportProgress: boolean = false ): Observable<any> {
+
+        if (buildId === null || buildId === undefined) {
+            throw new Error('Required parameter buildId was null or undefined when calling getBuildArtifactList.');
         }
 
-        if (file !== undefined) {
-            formParams = formParams.append('file', <any>file) || formParams;
+
+
+
+
+
+
+
+
```

Comment:
nit: intresting whitepace here

Reply:
Yeah, same as before it's an artifact of the swagger code generation.
Possibly a stylistic choice by them 🤷🏻‍♂️

Reply:
Correct. They've a bunch of if-statements in their templates, such as:

```mustache
{{ if foo }}
  foo!
{{ end }}
{{ if bar }}
  bar!
{{ end }}
{{ if moo }}
  moo!
{{ end }}
```

Which if none of `foo`, `bar`, or `moo` is set, then it results in 3 empty lines. One for each newline between the `{{ end }}` and next `{{ if }}` statements.